packages/core/src/subgraph/config.ts

```diff
@@ -0,0 +1,9 @@
+import 'cross-fetch/polyfill';
```

Comment:
what is cross-fetch for?

Reply:
I was having some problems for run the Apollo Client in node.js (without a browser) and found the answer here: https://stackoverflow.com/questions/50688998/using-apolloclient-with-node-js-fetch-is-not-found-globally-and-no-fetcher-pas

basically, to use the apollo client you need to polyfill or provide a fetch method

Reply:
:+1: 
add a comment with that, please